### DISCUSSION TITLE
Push notifications for setting Home mode on station doesn't work. Bropat asked for more logging. But I don't know how to deliver it.

### DISCUSSION THREAD

**Discussion by schliemann** [samemory/homebridge-eufy-security-control#42] [Ideas]
I opened an issue in the repo of the client library - https://github.com/bropat/eufy-security-client/issues/14#issuecomment-859277656

Bropat asked for the full log. I'm not quite sure how I can deliver this however. Obviously there is logging in the client library. I guess I could make my own fork of eufy-security-client and add a bunch of logging.

What do you guys think?

**Comment by lenoxys** [comment]
It could be tricky to maintain after that, does bropat has any recommendations to do the full log ?

**Comment by schliemann** [comment]
Yeah. He replied. Will add verbose logging on my machine.

**Comment by schliemann** [comment]
Got the trace log and sent it to bropat.
